test/tests/functions/baselinecorrection/chichi_A0126300_N.i

```diff
@@ -0,0 +1,102 @@
+# A test of the BaselineCorrection function applied to an earthquake
```

Comment:
Having many tests is not necessarily bad. But, do you think you can consolidate the number of test without sacrificing much the comprehensiveness?

Reply:
@somu15 yes

Reply:
@somu15 consider taking a moment to read my MooseDocs page. The tests each demonstrate a different aspect of this code. But, maybe there's someway I can consolidate all of the `mooseError()` tests into a single input file.